test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1006,6 +1006,26 @@ def forward(self, x):
         x = torch.randn(3, 4)
         self.run_test(ClampMaxModel(), x)
 
+        class ClampTensorModel(torch.nn.Module):
```

Comment:
Would it be better to move the following new cases to `test_clamp_dyn`, since it is for testing against tensor min max?

I'm surprised that the current test passed in CI for onnx opset < 11. ONNX::Clip under opset 11 accepts `min` and `max` as `attribute`, meaning as fixed static number. So theoretically it shouldn't be able to work with min/max being `torch.Tensor`. Could you check and paste the exported onnx IR graph to help clarify this? You can retrieve the graph by setting the flag `verbose=True` for `torch.onnx._export` api.